include/libopencm3/stm32/h7/pwr.h

```diff
@@ -0,0 +1,93 @@
+/** @defgroup pwr_defines PWR Defines
+
+@brief <b>Defined Constants and Types for the STM32H7xx Power Control</b>
+
+@ingroup STM32H7xx_defines
+
+@version 1.0.0
+
+@author @htmlonly &copy; @endhtmlonly 2019 Brian Viele <vielster@allocor.tech>
+
+@date 6 November 2019
+
+LGPL License Terms @ref lgpl_license
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2017 Matthew Lai <m@matthewlai.ca>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef LIBOPENCM3_PWR_H
+#define LIBOPENCM3_PWR_H
+
+/**@{*/
+
+/** @defgroup pwr_registers PWR Registers
+ * @ingroup STM32H_pwr_defines
+@{*/
+/** Power control register (PWR_CR1) */
```

Comment:
it's written now, but I don't generally see any point in having the PWR_CR1 _again_ in the comment personally :)